watcher/local/statespubsub.go

```diff
@@ -52,9 +57,13 @@ func newStatesPubSub() *statesPubSub {
 // client to publish transactions in correct order and ensure they are the
 // valid.
 //
+// Context is added as this method implements watcher.StatesPub interface.
+// However, is is not used as there no network communications involved in local
+// watcher.
+//
```

Comment:
```suggestion
```

I don't think you need to mention this. It becomes apparent when reading the code.